go.mod

```diff
@@ -5,30 +5,33 @@ go 1.12
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/DATA-DOG/go-sqlmock v1.3.3
+	github.com/coreos/bbolt v1.3.3 // indirect
+	github.com/coreos/etcd v3.3.13+incompatible // indirect
 	github.com/coreos/go-semver v0.3.0
 	github.com/go-sql-driver/mysql v1.4.1
 	github.com/gogo/protobuf v1.3.1
+	github.com/golang/lint v0.0.0-20180702182130-06c8688daad7 // indirect
 	github.com/golang/mock v1.3.1
+	github.com/golang/snappy v0.0.1 // indirect
 	github.com/joho/sqltocsv v0.0.0-20190824231449-5650f27fd5b6
-	github.com/pingcap/check v0.0.0-20190102082844-67f458068fc8
+	github.com/myesui/uuid v1.0.0 // indirect
+	github.com/pingcap/check v0.0.0-20191107115940-caf2b9e6ccf4
 	github.com/pingcap/errors v0.11.4
 	github.com/pingcap/failpoint v0.0.0-20190708053854-e7b1061e6e81
-	github.com/pingcap/kvproto v0.0.0-20191011042334-8ee4fd8fb4ca
-	github.com/pingcap/log v0.0.0-20190307075452-bd41d9273596
-	github.com/pingcap/parser v0.0.0-20190910041007-2a177b291004
-	github.com/pingcap/tidb v1.1.0-beta.0.20190929123532-694e086e7914
-	github.com/pingcap/tidb-tools v3.0.4+incompatible
+	github.com/pingcap/kvproto v0.0.0-20191202044712-32be31591b03
+	github.com/pingcap/log v0.0.0-20191012051959-b742a5d432e9
+	github.com/pingcap/parser v0.0.0-20191112053614-3b43b46331d5
+	github.com/pingcap/tidb v0.0.0-20191115021711-b274eb2079dc
```

Comment:
why is the TiDB version set to pingcap/tidb@b274eb2079dc? It is neither the latest master nor latest release-3.0 (may be related to the build failure?)

Reply:
build failure is caused by go1.12, although `b274eb2079dc` is not the latest version, it still need go1.13 to build.

Reply:
build failure is caused by go1.12, although b274eb2079dc is not the latest version, it still need go1.13 to build.